unit_tests/src/test_misc.cpp

```diff
@@ -0,0 +1,21 @@
+#include <catch.hpp>
+
+#include "version.hpp"
+
+TEST_CASE("version equivalence operators")
+{
+    constexpr irods::version v4290{4, 2, 9};
+
+    REQUIRE(v4290 == v4290);
+    REQUIRE(v4290 <= v4290);
+    REQUIRE(v4290 >= v4290);
+
+    constexpr irods::version v4300{4, 3};
+
+    REQUIRE(v4290 != v4300);
+    REQUIRE(v4290 <  v4300);
+    REQUIRE(v4290 <= v4300);
+    REQUIRE(v4300 >  v4290);
+    REQUIRE(v4300 >= v4290);
+}
```

Comment:
Maybe this could be `test_version.cpp`? `test_misc` is unclear to me (is it intended for `rcMisc.h`?)

Reply:
rename to test_version.cpp